components/versionreporting/02-variabes.tf

```diff
@@ -6,18 +6,10 @@ variable "environment" {
   default = "stg"
 }
 
-variable "product" {
-  default = "sds-platform"
-}
-
```

Comment:
Can we keep product name? May cause tagging errors / confusion

Reply:
I see var.serviceName in common tags but struggling to see where defined?

Reply:
resolved